### DISCUSSION TITLE
Frigate as a backup to Eufy

### DISCUSSION THREAD

**Discussion by MrCaspan** [blakeblackshear/frigate#9167] [Ask A Question]
I have decided to use Frigate only as a backup for my Eufy system. So all I need Frigate to do is provide storage of 24x7 recordings of each camera. When I do this and there are no longer recorded events, so when i look at a cameras recording all I get is 1 hour long options on the cameras recording page. This is great...I can easily click the hour I want and scrub along. BUT there is no way to download this recording when it is not an event. So I figurered I would go to the source and create a SMB share to my Frigate footage directory to get a copy of the recording as Frigate does store it in a somewhat easy to use structure, but there are 2 Issues.:

1 All the folders are in UTC time and makes it really painful to find your footage. I do understnad why this is done for daylight savings issues that would overwrite the 2AM folder on days that the time is set back 1 hour. but it does make footage hard to find unless you know your current time zone offset.. but hey its fine the footage is there you jsut need to do a little math :)

2. All the recordings are only 10 seconds long. Again I get why this is, a corrupt stream or issue will only lose you 10 seconds of footage insted of an hour but this last issue bring me to my actual question!

HOW, (if you don't use events recordings) can you download from the WebUI a copy of the hour footage? Again I only need Frigate for backup purposes and dont need to wasted CPU cycles on event detection or storage!

Hopefully there is a way to do this that I have just missed. And if there is not a way to do this maybe provide some options for a tool that could stitch these files into an hour segment for me?

**Comment by NickM-27** [comment]
In frigate 0.13 recordings can be exported using the ui, integration, api.

**Comment by MrCaspan** [comment]
Oh that is amazing.. I am assuming 0.13 is still in beta? Im on TrueNAS so it might be a bit before we see thi as an iX app but thats great news!!

**Reply by NickM-27** [reply]
It was moved to RC today

**Comment by MrCaspan** [comment]
NICE!!! Thanks Nick!

**Comment by MrCaspan** [comment]
If I can make a suggestion.. after watching all the videos on how this is implemented. It still seems very manual that you have to go to an export section. you have to tell it where to start where to end. It would be very useful that on the recording page for the camera beside the hours that you just put a little download image or icon. This way when you're watching a recording. you can easily press this button to download that hour of footage. It would use the same back end but having the link there already instead of having a user manually look at the recording. it was on January 1st between 7:00 p.m. and 8:00 p.m. Then going to the export page then manually selecting all that information on a form again. It would be very useful to just have it right there even if all it does is take you to the export form and just pre-fills it out for you.. and then that way you can select things like time lapse, not time lapse or shorten it to only be 30 minutes. I feel like this would be a very simple change using existing system already and make it much more user friendly, which I'm assuming is the point!.

like I know on 0.12 backend mechanism is already there. you Just have to know how to format the URL using epoch time etc.. So this should just be a front end using a little bit of code to figure out what each one of these links should be.. I know it might be a little bit more difficult to take you to the export page and prefilling all the info so as  a stopped gap. let us easily have a link to download the hour for now!

**Reply by NickM-27** [reply]
The ui is being rewritten in 0.14 so something like this may be implemented at that point, 0.13 will not have any more changes except fixes

**Reply by MrCaspan** [reply]
awesome thanks. I think I'll write a simple html /JS injector and make it available to the community so that those that want this can have something easier till then using a Chrome plugin
